ydb/core/fq/libs/actors/pending_fetcher.cpp

```diff
@@ -181,7 +181,7 @@ class TPendingFetcher : public NActors::TActorBootstrapped<TPendingFetcher> {
         , Monitoring(monitoring)
         , ComputeConfig(config.GetCompute())
         , S3ActorsFactory(std::move(s3ActorsFactory))
-        , DefaultPqGateway(std::move(defaultPqGateway))
+        , PqGatewayFactory(pqGatewayFactory)
```

Comment:
very nit: `std::move()` (it's more about consistency; unless you need another round -- feel free to ignore)